docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
       retries: 3
       start_period: 5s
   trillian-log-server:
-    image: gcr.io/trillian-opensource-ci/log_server:v1.3.13
+    image: gcr.io/trillian-opensource-ci/log_server:5e12fb368c8fd19e10aeb5a5cf785107f8069c08
```

Comment:
OK, so actually I think we need to split this change up into two so we're safe. It's good you tested with these containers, but we need to also prove that the code works with the old containers so we can roll forward/back safely.

So could you drop this part from this PR, when tests pass we'll merge, then we send another PR to bump this part, when tests pass there we then roll out in prod?